doc/building.md

```diff
@@ -546,6 +546,20 @@ Hotspot.
 Use `--with-libffi=<path>` if `configure` does not properly locate your libffi
 files.
 
+### libsleef
```

Comment:
You will need to regenerate building.html as well. `make update-build-docs` using pandoc 2.19.2.